packages/opentelemetry-tracing/src/config.ts

```diff
@@ -32,7 +32,6 @@ export const DEFAULT_MAX_LINKS_PER_SPAN = 32;
 export const DEFAULT_CONFIG = {
   defaultAttributes: {},
   logLevel: LogLevel.INFO,
-  sampler: ALWAYS_SAMPLER,
```

Comment:
I think we should keep always sampler as default and if `OTEL_SAMPLING_PROBABILITY` env is set, use `ProbabilitySampler`. WDYT?

Also, please take a look https://github.com/open-telemetry/opentelemetry-js/pull/1279.

Reply:
So I'm a bit confused about always sampler, I guess its a spec thing, but it really just in a probability sampler with a a probability of 1 right? This is the default for `ProbabilitySampler(getEnv().OTEL_SAMPLING_PROBABILITY)` i.e. if the env is unset, it will default to 1.

We can go the route of explicitly using the always sampler when no env is set, but then i wonder if the default value for the getEnv is a legit safeguard or maybe its just redundant/clutter?

Reply:
Sure sgtm. I'll adjust accordingly.

Reply:
`parentOrElse` is pretty cool, today we are putting a low probability across the board, but perhaps we want to pass that low probability to a parentOrElse 🤔 🤔 

Should OTEL_SAMPLING_PROBABILITY now adjust probability in the delagatedSampler of a parentOrElse sampler?? is parentOrElse always better?

Reply:
OTEL_SAMPLING_PROBABILITY should probably be read by the probability sampler and the parent or else sampler. The parent or else sampler will use the always on sampler by default, but if the OTEL_SAMPLING_PROBABILITY env var exists and is the right format, it could use the probability sampler as a delegate. No need for it to play with probabilities because the probability sampler will just reread the env var.

Reply:
alright, so my pending question is:

if you set OTEL_SAMPLING_PROBABILITY, which sampler does the tracer user? a parentOrElse or a probabilitySampler (directly)?

Reply:
Oh you are saying if you have coded in the use of a parentOrElse sampler, and left the delegation empty, `OTEL_SAMPLING_PROBABILITY` will place a probability sampler in it?

Reply:
There is no such thing as a parent or else sampler with an empty delegate

Reply:
>if you set OTEL_SAMPLING_PROBABILITY, (without any further config) which sampler does the tracer user? a parentOrElse or a probabilitySampler (directly)?

it should use a parent or else sampler with a probability delagate

Reply:
thanks, just to confirm: parentOrElse with a prob delegate is always favored over a direct prob sampler?

Reply:
Unless specifically configured otherwise by the user, yes.